### PR TITLE
Event examples

### PR DESCRIPTION
**Pull request by mbowergenesys**

### CODE REVIEW

File: src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleHelper.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    
+    stateUpdate: function(component, action) {
+        var clientOrigin = component.get('v.clientOrigin'); 
+        var source = component.get('v.postMessageSource');
+        var id = component.get('v.interactionId');
+        
+        if(source) {
+            source.postMessage({
+                type: 'PureCloud.User.updateState',
```
Comment (junji-genesys) [inline src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleHelper.js]: I noticed this as I was installing it and playing around with it, but these two event types looks very similar:
- PureCloud.User.updateStatus
- PureCloud.User.updateState
And both of them are under PureCloud.User namespace.
So my thought was, Does it make sense to use something like this for interaction state update?
- PureCloud.Interaction.updateState
Also, in PEF actions, it's defined under Interaction namespace:
image
Any thought?

Comment (mbowergenesys) [reply src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleHelper.js]: I think you're right, it should PureCloud.Interaction. Good catch!

File: src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleController.apxc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    @AuraEnabled
+    public static String getCallCenterUrl() {
+        String url = '';
+        
+        String userId = UserInfo.getUserId();
+        User userRecord = [Select CallCenterId From User Where Id =: userId Limit 1];
+        
+        if(String.isNotBlank(userRecord.CallCenterId)) {
+            CallCenter callCenterRecord = [Select AdapterUrl From CallCenter Where Id =: userREcord.CallCenterId Limit 1];
```
Comment (kristenoliphant) [inline src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleController.apxc]: The casing is off. userREcord should be userRecord.

File: src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleController.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+({  
+    initComponent: function(component, event, helper) {
```
Comment (kristenoliphant) [inline src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleController.js]: In here did we also want to add an example of how to subscribe to the 3 different subscription types and display their events in a UI similar to how we do it for classic console?

Comment (kristenoliphant) [reply src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExampleController.js]: After refreshing myself with the code I guess there's not an example of subscribing to the events but more so just outputting the different events in a UI.

File: src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExample.cmp [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<aura:component implements="flexipage:availableForAllPageTypes" access="global" controller="PCSalesforceEventsExampleController">
+    <lightning:utilityBarAPI aura:id="utilitybar" />
+    <lightning:empApi aura:id="empApi"/>
```
Comment (kristenoliphant) [inline src/Lightning Experience/events-in-salesforce/PCSalesforceEventsExample.cmp]: It doesn't look like this is being used.

File: src/Classic console/events-in-salesforce/readme.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+### Create a Custom Console Component
+
+1. In Salesforce, click **Setup**.
+2. Search for and click **Custom Console Components**.
+3. Click **New**.
+4. Add names to the **Name** and **Button Name** text boxes.
+5. Under **Component**, search for and select the Visualforce page that you created.
+6. Click **Save**.
+
+### Select Your Visualforce Page // <-- add your visualforce page to the component? >
```
Comment (kristenoliphant) [inline src/Classic console/events-in-salesforce/readme.md]: FYI There's a question about the title of this section. At this point you are not doing anything with the Visualforce page directly. You are adding the Custom Console Component to an App.

**Pull request merged**